Makefile

```diff
@@ -0,0 +1,24 @@
+CC := gcc
+
+INC_DIR := inc
+SRC_DIR := src
+BUILD_DIR := build
+
+HEADERS := $(wildcard $(INC_DIR)/*.h)
+SOURCES := $(wildcard $(SRC_DIR)/*.c)
+OBJECTS := $(SOURCES:$(SRC_DIR)/%.c=$(BUILD_DIR)/%.o)
+
+__dir := $(shell mkdir -p $(BUILD_DIR))
+
+DEBUG = -g
+CFLAGS = -Wall -I$(INC_DIR) $(DEBUG)
+LDFLAGS = -lpthread
```

Comment:
We could add `-lncursesw` here

Reply:
Sure! I will refer to yours and add the flag. Thanks.